common/source/docs/common-optional-hardware.rst

```diff
@@ -47,6 +47,7 @@ information related to Autopilot selection see :ref:`Autopilot Hardware Options
     Landing Gear/ Retractable Camera Mount <common-landing-gear>
 [/site]
     LEDs (external) <common-external-leds>
+    Multiple Radio Control Receivers <common-multiple-rx>
```

Comment:
I wonder if the title should be "Radio Control Receivers (Multiple)" to keep the "Radio Control" part at the front.  I think it's not important for now but maybe we will eventually decide that we should have a section on RC Receivers.